functions/src/requestCreated/index.js

```diff
@@ -15,6 +17,7 @@ const pubSubClient = new PubSub();
 /**
  * Send FCM message to user about request being created
  * @param {string} userId - Id of user to send FCM message
+ * @returns {Promise} Resolves with message id
  */
 async function sendFcmToUser(userId) {
   const messageObject = { userId, message: 'Request Created' };
```

Comment:
Can we make the `message` more meaningful, and the notification more functional where by clicking on the notification we get taken to the request?

Reply:
What are we thinking the message should say? Something like:
```
`Request for ${needs.join(', ')} near ${generalLocationName}` // i.e Request for grocery-pickup near Madison